### ISSUE TITLE
Embed our Twitter feed

### ISSUE THREAD

**Issue by karol-majewski** [WrocTypeScript/website#24]
To embed tweets by @WrocTypeScript:

```html
<a class="twitter-timeline" href="https://twitter.com/WrocTypeScript?ref_src=twsrc%5Etfw">Tweets by WrocTypeScript</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

It's also possible to display other things.

![image](https://user-images.githubusercontent.com/20233319/50035683-dc27ab80-0003-11e9-9648-182c62a0002d.png)

See https://publish.twitter.com

**Comment by hasparus** [comment]
Możemy dodać `data-tweet-limit`
```
<a
   class="twitter-timeline"
   href="https://twitter.com/wroctypescript"
   data-tweet-limit="1"
>
  Tweets by @twittername
</a>
<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

**Comment by hasparus** [comment]
closed by #34

**Comment by karol-majewski** [comment]
> Edit: It didn't.

OP did deliver :D
